diagnostic.md

```diff
@@ -4,6 +4,15 @@ Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
 <!-- your answer starts here -->
+I'm not sure what the question is asking.  Do you want code to select four
+different elements using jQuery?  When I type $() in the console, I get "$ is not
+a function" - does this mean that jQuery is not loaded? I see that window.jQuery
+is "undefined." Maybe I missed how to load it into a page.  Maybe I haven't had
+enough coffee.
+$(".container")
+$("div")
+$("#facebox")
+$(".js-touch-events")
```

Comment:
Don't overthink this one, you got it right. We were simply looking for four selectors on the page that you _could_ select using jQuery